lib/trino-parquet/src/main/java/io/trino/parquet/reader/ParquetReader.java

```diff
@@ -348,4 +456,28 @@ public AggregatedMemoryContext getSystemMemoryContext()
     {
         return systemMemoryContext;
     }
+
+    private static <T> List<T> listWithNulls(int size)
+    {
+        return Stream.generate(() -> (T) null).limit(size).collect(Collectors.toCollection(ArrayList<T>::new));
```

Comment:
Format like this for readability:
```suggestion
        return Stream.generate(() -> (T) null)
                .limit(size)
                .collect(Collectors.toCollection(ArrayList<T>::new));
```